src/components/form/SelectCondition/index.tsx

```diff
@@ -1,29 +1,19 @@
 import { useDebounceCallback } from '@react-hook/debounce'
-import { SelectConditionModal } from 'components/modals/SelectConditionModal'
-import { Error, ErrorContainer } from 'components/pureStyledComponents/Error'
-import { Row } from 'components/pureStyledComponents/Row'
-import { Textfield } from 'components/pureStyledComponents/Textfield'
-import { TitleControl } from 'components/pureStyledComponents/TitleControl'
-import { TitleValue } from 'components/text/TitleValue'
 import { useConditionContext } from 'contexts/ConditionContext'
 import React from 'react'
-import styled from 'styled-components'
 
-const InfoMessage = styled.p`
-  color: ${(props) => props.theme.colors.lightGrey};
-  font-size: 14px;
-  font-weight: 400;
-  font-style: italic;
-  line-height: 1.4;
-  margin: 0 0 5px 0;
-  text-align: left;
+import { SelectConditionModal } from '../../modals/SelectConditionModal'
```

Comment:
Lets use all absolute or all relative paths

Reply:
Yeah. We have a linter rule for sorting and grouping imports and such. Absolute paths are messing with that.